version.php

```diff
@@ -30,8 +30,8 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2023101905;
-$plugin->release   = 2022101905;
+$plugin->version   = 2023101906;
```

Comment:
Looks like the year was bumped. This I believe should be a microbump. 

Once this is fixed it should be good to go.

Reply:
ah good catch! updating....

Reply:
Actually I think this is correct, the version is 1 yr ahead of the release, these diverged ~ 1yr ago in https://github.com/catalyst/moodle-mod_facetoface/commit/c27e172247dfa3f5fe1128d1f78934db7f6a3bbd

Reply:
Oh that's very strange. Since release doesn't actually do anything, can you make it equivalent for now to keep it generally consistent for our plugins, and to prevent it raising flags in a future review? I think it's fine to fix it now.

Reply:
Yep agreed - updated